src/features/hotspots/settings/discovery/DiscoveryModeResults.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { memo, useEffect, useMemo, useState } from 'react'
+import { Hotspot } from '@helium/http'
+import { useTranslation } from 'react-i18next'
+import { isEqual, startCase } from 'lodash'
+import haversine from 'haversine-distance'
+import { useSelector } from 'react-redux'
+import Box from '../../../../components/Box'
+import DiscoveryMap, {
+  MapSelectDetail,
+  ANIM_LOOP_LENGTH_MS,
+} from './DiscoveryMap'
+import {
+  DiscoveryRequest,
+  DiscoveryResponse,
+  DISCOVERY_DURATION_MINUTES,
+} from '../../../../store/discovery/discoveryTypes'
+import animateTransition from '../../../../utils/animateTransition'
+import { hp } from '../../../../utils/layout'
+import { RootState } from '../../../../store/rootReducer'
+import { useAppDispatch } from '../../../../store/store'
+import { fetchNetworkHotspots } from '../../../../store/networkHotspots/networkHotspotsSlice'
+import { getHotspotDetails } from '../../../../utils/appDataClient'
+import DiscoveryModeResultsCard from './DiscoveryModeResultsCard'
+import { usesMetricSystem } from '../../../../utils/i18n'
+
+type Props = {
+  request?: DiscoveryRequest | null
+  hotspot: Hotspot
+  isPolling: boolean
+  requestTime: number
+  currentTime: number
+}
+const DiscoveryModeResults = ({
+  request,
+  hotspot,
+  isPolling,
+  currentTime,
+  requestTime,
+}: Props) => {
+  const { t } = useTranslation()
+  const [deDupedResponses, setDeDupedResponses] = useState<DiscoveryResponse[]>(
+    [],
+  )
+  const [overlayDetails, setOverlayDetails] = useState<
+    {
+      distance: string
+      rewardScale: string
+    } & MapSelectDetail
+  >()
+  const dispatch = useAppDispatch()
+  const networkHotspots = useSelector(
+    (state: RootState) => state.networkHotspots.networkHotspots,
+    isEqual,
+  )
+
+  const hotspotCoords = useMemo(() => {
+    return [hotspot.lng || 0, hotspot.lat || 0]
+  }, [hotspot.lat, hotspot.lng])
+
+  useEffect(() => {
+    const oneMileInDegrees = 1 / 69 // close enough => depends on your location. It's 68.7 at the equator and 69.4 at the poles, but yolo
+    const offset = 15 * oneMileInDegrees // TODO: 15 mile offset. Is this adequate?
```

Comment:
So this is just to display network hotspots right? But some hotspots may still be found outside of the 15 mile range?